tests/test_adapters.py

```diff
@@ -62,6 +64,7 @@ def test_adapter_indexes(adapter, original, value, target, expected, mocker):
         (Ipca15, 312, date(1994, 1, 1), date(2019, 12, 1)),
         (IpcaE, 312, date(1994, 1, 1), date(2019, 12, 1)),
         (Selic, 300, date(1995, 1, 1), date(2019, 12, 1)),
+        (Cpiaucsl, 876, date(1947, 1, 1), date(2019, 12, 1)),
```

Comment:
Can we keep these decorator lines in alphabetical order for better maintainability?